translations/ui-data-import/en.json

```diff
@@ -50,9 +55,15 @@
   "settings.other": "Other",
   "settings.fileExtensions": "File extensions",
 
-  "modal.header": "Inconsistent file extensions",
-  "modal.message": "You cannot upload files with {highlightedText}. Please upload files with the same extension.",
-  "modal.messageHighlightedText": "different extensions",
-  "modal.cancel": "Cancel",
-  "modal.actionButtonText": "Choose other files to upload"
+  "modal.fileExtensions.header": "Inconsistent file extensions",
+  "modal.fileExtensions.message": "You cannot upload files with {highlightedText}. Please upload files with the same extension.",
+  "modal.fileExtensions.messageHighlightedText": "different extensions",
+  "modal.fileExtensions.cancel": "Cancel",
+  "modal.fileExtensions.actionButtonText": "Choose other files to upload",
+
+  "modal.leavePage.header": "Leave the page",
+  "modal.leavePage.message": "There are {highlightedText}. If you would like to complete all uploads, press the \"Keep uploading\" button.",
+  "modal.leavePage.messageHighlightedText": "unfinished uploads",
+  "modal.leavePage.cancel": "Close & lose partial uploads",
+  "modal.leavePage.actionButtonText": "Keep uploading"
```

Comment:
Let's avoid using `Text` ending. Eventually, everything within this file is text.